mongo/mongo.go

```diff
@@ -134,3 +138,24 @@ func (m *Mongo) UpdateJobState(id, newState string) (err error) {
 	_, err = s.DB(m.Database).C(m.Collection).Upsert(bson.M{"id": id}, update)
 	return
 }
+
+func (m *Mongo) Close(ctx context.Context) error {
```

Comment:
Is this function needed anymore? Should it be replaced with the go-ns function.

Reply:
now replaced with go-ns/mongo